### ISSUE TITLE
Permissions issues with electron sandbox on archlinux

### ISSUE THREAD

**Issue by tulsileathers** [beakerbrowser/beaker#1428]
Operating System: Archlinux (Manjaro)

Beaker version: 0.8.8

Hi all, 
I'm trying to build Beaker from source and having some issues. 
First I run `npm install` which completes successfully
then `npm run rebuild` also successful
then `npm start` here's where i first get my error
```
[8958:0619/111505.829308:FATAL:setuid_sandbox_host.cc(157)] The SUID sandbox helper binary was found, but is not configured correctly. Rather than run without sandboxing I'm aborting now. You need to make sure that ./node_modules/electron/dist/chrome-sandbox is owned by root and has mode 4755.
```
a chmod and chown later `npm start` now succeeds and Beaker starts

so now I successfully build the AppImage with `npm run release`
but when running the built AppImage I get the same error but this time pointing at a temporary chrome-sandbox

```
You need to make sure that /tmp/.mount_BeakerhN8lvV/chrome-sandbox is owned by root and has mode 4755.
```

I can't however change that file's permissions as it seems to be deleted after the app crashes. Any suggestions? I'm not super familiar with electron so maybe I'm missing something obvious?

**Comment by brechtcs** [comment]
I've noticed something similar on Debian 10. Haven't been able to research it in detail though.

**Comment by pfrazee** [comment]
Thanks for filing. That's a pain! I'll check into it.

**Comment by tulsileathers** [comment]
@pfrazee let me know if I can help in anyway!

**Comment by pfrazee** [comment]
Thanks! I just need to dig into electron docs and issues and look for some pointers.

**Comment by vaga007** [comment]
Facing similar issue (one with temporary path ) on CentOS7 as well.

**Comment by raphaelbastide** [comment]
I confirm this issue on Manjaro (Archlinux). The fix #1524 that works for me on Debian `sudo sysctl kernel.unprivileged_userns_clone=1`returns the following error on Manjaro: `sysctl: cannot stat /proc/sys/kernel/unprivileged_userns_clone: No such file or directory`
I am not able to install the last appimages on Manjaro linux.
